### DISCUSSION TITLE
Scheudle for the Services

### DISCUSSION THREAD

**Discussion by smartonaut** [AdguardTeam/AdGuardHome#7347] [Ideas]
It would be great if you could switch the services on and off according to a schedule, so that you can restrict your children a little more. Like activating and deactivating game servers at regular times. Discord and other services

**Comment by Dynamic5912** [comment]
> It would be great if you could switch the services on and off according to a schedule, so that you can restrict your children a little more. Like activating and deactivating game servers at regular times. Discord and other services

You can do that on a per-client basis already...

**Comment by smartonaut** [comment]
I have the OPN sense in front of my network, all requests go via ADblock Home. And I want the children to be able to surf, but I can now say that they can surf from 10 p.m. to 3 p.m. but not play Valorent, for example. 
Is that possible? since all requests come from the OPNsense?

**Reply by Dynamic5912** [reply]
Yes. If AGH is your LAN DNS for your devices
